models/demos/wormhole/llama31_8b_N300/tt/model_config.py

```diff
@@ -0,0 +1,440 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import ttnn
+from pathlib import Path
+from loguru import logger
+import torch
```

Comment:
We import torch but I don't see where we actually need it.

Reply:
We use it in the `load_state_dict` function.